packages/node_modules/@ciscospark/widget-space/src/enhancers/setup.js

```diff
@@ -30,14 +37,29 @@ function setup(props) {
     && sparkState.get('registered')
     && !sparkState.get('hasError')) {
     if (!widgetStatus.isFetchingSpaceDetails && !errors.get('hasError') && !spaceDetails) {
-      if (spaceId) {
-        props.getSpaceDetails(sparkInstance, spaceId, props.intl);
+      // If the selector isn't returning a destination object and we have props for them, store
```

Comment:
Heavy lifting of converting legacy destination props to new destination type and id here